src/main/resources/application.yml

```diff
@@ -133,6 +133,7 @@ hmpps.s3:
   buckets:
     document-management:
       bucketName: ${S3_DOCUMENT_BUCKET_NAME}
+      maxFileSize:
```

Comment:
is this incomplete?

Reply:
redundant and not required. it is already specified as part of spring config in the same.